### PR TITLE
Build for 2021

### PR DESCRIPTION
**Pull request by rtzoeller**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-custom-device-development-tools/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Drop 2017 and add 2021 builds.

### Why should this Pull Request be merged?

We are slowly bringing the 2021 toolchain online, and this repo is a dependency of many others.

### What testing has been done?

Relying on PR build.

### CODE REVIEW

**Review by Karl-G1 — APPROVED** [review]

**Review by buckd — COMMENTED** [review]
This all looks fine. Based on the build failure, I'm anticipating some other change, so I'll wait until there's a successful build to approve.

**Comment by rtzoeller** [comment]
@buckd I manually bootstrapped the build machine by copying over a previous `vi.lib/NI/` directory. The build is now passing.

**Review by buckd — APPROVED** [review]

**Pull request merged**
